pkg/build/graph/analysis/bc.go

```diff
@@ -4,30 +4,38 @@ import (
 	"fmt"
 	"sort"
 	"strings"
+	"time"
 
 	"github.com/gonum/graph"
 	"github.com/gonum/graph/topo"
 
+	"k8s.io/kubernetes/pkg/api/unversioned"
+
 	osgraph "github.com/openshift/origin/pkg/api/graph"
 	buildapi "github.com/openshift/origin/pkg/build/api"
 	buildedges "github.com/openshift/origin/pkg/build/graph"
 	buildgraph "github.com/openshift/origin/pkg/build/graph/nodes"
+	imageapi "github.com/openshift/origin/pkg/image/api"
 	imageedges "github.com/openshift/origin/pkg/image/graph"
 	imagegraph "github.com/openshift/origin/pkg/image/graph/nodes"
 )
 
 const (
-	TagNotAvailableWarning     = "ImageStreamTagNotAvailable"
-	LatestBuildFailedErr       = "LatestBuildFailed"
-	MissingRequiredRegistryErr = "MissingRequiredRegistry"
-	MissingImageStreamErr      = "MissingImageStream"
-	CyclicBuildConfigWarning   = "CyclicBuildConfig"
+	TagNotAvailableWarning         = "ImageStreamTagNotAvailable"
+	LatestBuildFailedErr           = "LatestBuildFailed"
+	MissingRequiredRegistryErr     = "MissingRequiredRegistry"
+	MissingOutputImageStreamErr    = "MissingOutputImageStream"
+	CyclicBuildConfigWarning       = "CyclicBuildConfig"
+	MissingImageStreamTagWarning   = "MissingImageStreamTag"
```

Comment:
@gabemontero yes. Not sure if others ( @deads2k @bparees ) agree with that distinction. 

When there is a bc pointing to a tag we currently warn with:
```go
"%s needs to be imported or created by a build"
```
and suggest `start-build` because we have a bc and no builds.

When there is no bc pointing to a tag, we could output
```go
"%s needs to be imported"
```
I would call that an error but I don't feel strong.